apps/frontend/web/src/components/Navbar.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  Box,
+  HStack,
+  Heading,
+  Center,
+  Image,
+  IconButton,
+  Text,
+  Grid,
+} from '@chakra-ui/react';
+import { ArrowRightIcon, ArrowLeftIcon } from '@chakra-ui/icons';
+import { useState } from 'react';
+const Navbar = () => {
+  const iconsSize: string = '30px';
+  const paddingBetweenIcons: string = '15px';
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <Box  w={`${isOpen ? '220px' : '60px'}`}>
+    <Box
+      boxShadow="inner"
+      bg="blue.200"
+      w={`${isOpen ? '220px' : '60px'}`}
+      h="calc(100vh)"
+      position="absolute"
+      top="0"
+      left="0"
+      overflow="scroll"
+    >
+      
+      <Box bg="blue.500" w={`${isOpen ? '220px' : '60px'}`} h="80px">
+        {isOpen && (
+          <Heading fontSize="xl" pt="25px">
+            DevManager
```

Comment:
Il est préférable pour des raisons de sécurité d'emballer les chaînes de caractères dans des `{""}`
```suggestion
            {"DevManager"}
```